plugin/src/templates/api/gatsbyFunction.ts

```diff
@@ -16,11 +13,6 @@ import {
   AugmentedGatsbyFunctionRequest,
 } from './utils'
 
-/**
- * Depending on the version of '@gatsbyjs/reach-router' installed, the 'match' method may not be defined.
- * This check ensures that this continues to work as expected between v1 and v2 of the package.
- */
-const reachMatch = reachRouterMatch || reachRouterMatchPath
```

Comment:
the behavior of `match` (v2 of router) / `matchPath` (v1 of router) did not change - with move to actual dependency we don't need to pick one or the other and can straight import `match as reachMatch`